src/components/video/Duration.vue

```diff
@@ -0,0 +1,49 @@
+<script setup lang="ts">
+import { useI18n } from 'vue-i18n';
+import { Icon } from '@iconify/vue/dist/iconify.js';
+
+import messages from '@/i18n/video';
+
+defineProps<{
+  start?: number;
+  end?: number;
+}>();
+
+const i18n = useI18n({
+  messages,
+  useScope: 'local',
+});
+const { t } = i18n;
+
+function minutesSecondsFromSeconds(seconds: number): string {
+  let minutes: number = Math.floor(seconds / 60);
+  let remainingSeconds: number = Math.floor(seconds % 60);
+
+  return numToString(minutes, 2) + ':' + numToString(remainingSeconds, 2);
+}
+
+function numToString(number_: number, minLength: number): string {
+  let numAsString = number_.toString();
+
+  while (numAsString.length < minLength) {
+    numAsString = '0' + numAsString;
+  }
+
+  return numAsString;
+}
```

Comment:
```suggestion
function numToString(number_: number, minLength: number): string {
  return String(number_).padStart(minLength, "0");
}
```

Reply:
also… `zeroPadNumber` or something like that is probably a better name

Reply:
Accepted and implemented both the code and the name change.